### ISSUE TITLE
[Feature Request] Clean Nozzle Between Sequential Objects

### ISSUE THREAD

**Issue by The-Lambs** [bambulab/BambuStudio#1931]
I didn't find anything on this specific item on the Bambu Labs forums.

I was just printing a plate with the print sequence set to "By Object", and I noticed that after printing one object which had an ironed surface, the print head moved immediately to start printing the brim of the next object in the sequence. But due to the ironing on the previous object, there was still some filament stuck to the outside of the print head, and this was integrated with the first layer on the next object, causing a defect.

I would like a setting to force a nozzle clean between objects when printing by object sequence.

**Comment by Majestic7979** [comment]
Oh I had this issue too in the past! Yes, definitely.

**Comment by eliezedeck** [comment]
Yes, even with very good settings, ironing tends to eventually create some left-over filaments on the head. Very desired feature indeed.

**Comment by reneroth** [comment]
bumping this because it's a really nice feature :)

**Comment by sosojni** [comment]
I will add my bump to it too. I was not ironing but i am printing 3 objects on 0.08 layer size and too much plastic oozes out of the hot end in between objects which reduces chance of successfully printing 2. or 3. object. Printer doing cleanup between objects would be good thing to have.
